src/transforms/babel.js

```diff
@@ -8,22 +8,47 @@ module.exports = async function(asset) {
 
   await asset.parseIfNeeded();
 
-  let config = {
+  let babelrc =
+    (await config.load(asset.name, ['.babelrc', '.babelrc.js'])) || {};
+  let babelConfig = Object.assign(babelrc, {
     code: false,
-    filename: asset.name
-  };
+    filename: asset.name,
+    babelrc: false
+  });
+
+  // TODO: Support .babelignore
```

Comment:
In order to set modules: false to babel-preset-env I had to disabled automatic loading of .babelrc which as well disables loading of .babelrcignore :(

Reply:
I'll add support for that if you are happy with everything else